empyrical/tests/test_stats.py

```diff
@@ -955,3 +971,159 @@ def test_cagr_noisy(self, returns, add_noise):
             cagr,
             noisy_cagr_2,
             1)
+
+    @property
+    def empyrical(self):
+        """
+        This returns a wrapper around the empyrical module, used a hook, so
+        tests perform, e.g. input conversions or return type checks, on each
+        call to an empyrical function. Each subclass can define this property
+        so that all the same tests are run but with different function inputs
+        or type checks.
+
+        This was done as part of enabling empyrical functions to work with
+        inputs of either pd.Series or np.ndarray, with the expectation that
+        they will return the same type as their input.
+
+        Returns
+        -------
+        empyrical
+
+        Notes
+        -----
+        Since some parameterized test parameters refer to attributes on the
+        real empyrical module at class body scope, this property must be
+        defined later in the body than those references. That way, the
+        attributes are looked up on the empyrical module, not this property.
+
+        e.g. empyrical.DAILY
```

Comment:
Could certainly do that.  Was trying to reduce the diff to show that the tests didn't change materially, but maybe it's confusing this way?